src/doc.ts

```diff
@@ -4,14 +4,14 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import * as ejs from 'ejs';
-import * as hljs from 'highlight.js';
 import * as marked from 'marked';
 import * as path from 'path';
 import * as string from 'underscore.string';
 import {Annotation} from './annotation';
 import {Docs} from './docs';
 import {Comment} from './dox';
 import {Options, Section} from './ts-helper';
+const hljs = require('highlight.js');
```

Comment:
Why is it needed?

Reply:
my vscode was complaining, maybe I used a wrong TypeScript version for my workspace. Now it's gone. Thank you Raymond for the update!